testprojects/src/protobuf/org/pantsbuild/testproject/import_from_buildroot/bar/baz.proto

```diff
@@ -0,0 +1,10 @@
+// Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
+// Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+package org.pantsbuild.testproject.import_from_buildroot;
+
```

Comment:
Good idea, added a README.md, since this is actually true of the entire package.